### DISCUSSION TITLE
Welcome to ipade Discussions!

### DISCUSSION THREAD

**Discussion by deleolajide** [inspired-futures/ipade#7] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by coderblaq0911** [comment]
Hi, I'm Mathew by name, A frontend developer. so i want to know the folder where you keep all the icons used in this project. Thanks

**Reply by deleolajide** [reply]
https://github.com/inspired-futures/ipade/tree/master/docs

**Reply by coderblaq0911** [reply]
Thank u sir,where exactly is the breakout room icon, sharescreen and the likes cos I couldn't find it inside the link sent.... Then where those icons are arranged... I'm finding it difficult to locate it and rearrange the icons to my taste sir.

**Reply by coderblaq0911** [reply]
I mean the toolbox icon folders

**Reply by deleolajide** [reply]
The additional icons are embeded SVGs. See https://github.com/inspired-futures/ipade/blob/035eda7fe953d9970aec24f5fd1aea10c8e0fa05/docs/custom_ofmeet.js#L9

**Reply by coderblaq0911** [reply]
Do u mean the arrangement of the icons is inside the custom_ofmeet.js????

**Reply by deleolajide** [reply]
Yes. This project only adds some extra icons. For all the icons, see https://github.com/jitsi/jitsi-meet

**Reply by coderblaq0911** [reply]
One more thing, I tried to add breakout room with recording on my own jitsi-meet code but it didn't work at all... can i know y??????????

**Comment by armandmorin** [comment]
Quick question what would I modify to use my own Jitsi install rather than the public one utilized here. Thanks.

**Reply by deleolajide** [reply]
Sorry, I don't use the standard Jitsi install. I use Jitsi with Openfire XMPP server. See the [Pade project](https://github.com/igniterealtime/openfire-pade-plugin)

**Reply by coderblaq0911** [reply]
ok sir.... but can i use ur codes to replicate mine?????

**Reply by deleolajide** [reply]
> ok sir.... but can i use ur codes to replicate mine?????

yes
